### PR TITLE
Display Cards

### PR DESCRIPTION
**Pull request by sebastienng**
Component displays Card for Cities and Trips. No links yet just static

### CODE REVIEW

File: src/components/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      <div className="h-4/5  w-[full] lg:w-[20rem]">
+        <img
+          src="https://www.parisinfo.com/var/otcp/sites/images/node_43/node_51/node_230/vue-a%C3%A9rienne-paris-tour-eiffel-coucher-de-soleil-|-630x405-|-%C2%A9-fotolia/19544352-1-fre-FR/Vue-a%C3%A9rienne-Paris-Tour-Eiffel-coucher-de-soleil-|-630x405-|-%C2%A9-Fotolia.jpg"
+          className="h-[16rem] lg:h-[18rem] object-cover rounded-xl shadow-lg shadow-gray-400/40 "
+          alt="bla"
+        />
+      </div>
+      {/* loc icon */}
+      <div className="flex items-center gap-1 lg:h-1/5 rounded-b-md pt-2">
+        <img src={iconLoc} alt="location point" className="w-[2rem] h-[2rem]" />
```
Comment (poollooo) [inline src/components/Card.jsx:18]: use w-8 instead, in tailwind each 1 increment is 0.25 rem, hover your tailwind css to get more informations about the actual css class

File: src/components/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import iconLoc from "../images/localisation-icon.png";
+
+const Card = (props) => {
+  const { title, desc } = props;
+  return (
+    <div className="flex flex-col gap-1 w-[20rem]">
+      {/* images */}
+      <div className="h-4/5  w-[full] lg:w-[20rem]">
```
Comment (poollooo) [inline src/components/Card.jsx:9]: use w-5 instead, in tailwind each 1 increment is 0.25 rem, hover your tailwind css to get more informations about the actual css class

File: src/components/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import iconLoc from "../images/localisation-icon.png";
+
+const Card = (props) => {
+  const { title, desc } = props;
+  return (
+    <div className="flex flex-col gap-1 w-[20rem]">
```
Comment (poollooo) [inline src/components/Card.jsx:7]: use w-5 instead, in tailwind each 1 increment is 0.25 rem, hover your tailwind css to get more informations about the actual css class

File: src/components/Card.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      {/* images */}
+      <div className="h-4/5  w-[full] lg:w-[20rem]">
+        <img
+          src="https://www.parisinfo.com/var/otcp/sites/images/node_43/node_51/node_230/vue-a%C3%A9rienne-paris-tour-eiffel-coucher-de-soleil-|-630x405-|-%C2%A9-fotolia/19544352-1-fre-FR/Vue-a%C3%A9rienne-Paris-Tour-Eiffel-coucher-de-soleil-|-630x405-|-%C2%A9-Fotolia.jpg"
+          className="h-[16rem] lg:h-[18rem] object-cover rounded-xl shadow-lg shadow-gray-400/40 "
```
Comment (poollooo) [inline src/components/Card.jsx:12]: same, use h-64

File: src/components/Card.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+          alt="bla"
+        />
+      </div>
+      {/* loc icon */}
+      <div className="flex items-center gap-1 lg:h-1/5 rounded-b-md pt-2">
+        <img src={iconLoc} alt="location point" className="w-[2rem] h-[2rem]" />
+        {/* TRIP info */}
+        <div>
+          <p className="font-bold text-s ">{title}</p>
```
Comment (poollooo) [inline src/components/Card.jsx:21]: text-s existe pas, c'est text-sm pour un small text

File: src/components/Card.jsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+          <p className=" text-s italic text-xs">{desc}</p>
```
Comment (poollooo) [inline src/components/Card.jsx:22]: 2 class tailwind pour du text, text-s existe pas donc y a pas de problème, mais normalement il devrait y avoir un warning

File: src/components/Card.jsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+      {/* images */}
+      <div className="h-4/5  w-[full] lg:w-[20rem]">
+        <img
+          src="https://www.parisinfo.com/var/otcp/sites/images/node_43/node_51/node_230/vue-a%C3%A9rienne-paris-tour-eiffel-coucher-de-soleil-|-630x405-|-%C2%A9-fotolia/19544352-1-fre-FR/Vue-a%C3%A9rienne-Paris-Tour-Eiffel-coucher-de-soleil-|-630x405-|-%C2%A9-Fotolia.jpg"
+          className="h-[16rem] lg:h-[18rem] object-cover rounded-xl shadow-lg shadow-gray-400/40 "
+          alt="bla"
+        />
+      </div>
+      {/* loc icon */}
+      <div className="flex items-center gap-1 lg:h-1/5 rounded-b-md pt-2">
```
Comment (poollooo) [inline src/components/Card.jsx:17]: rounded-b-md pas utiliser, clean useless code

**Review by poollooo — CHANGES_REQUESTED** [review]

**Review by poollooo — APPROVED** [review]

**Review by yuanliuddd — APPROVED** [review]

**Pull request merged**